### PR TITLE
Simplify binder example

### PR DESCRIPTION
**Pull request by anasmi**
Simply layout of the example where binder usage is shown. Move some parts to another example
Fix https://github.com/vaadin/vaadin-form-layout-flow/issues/57

### CODE REVIEW

File: vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java [hunk 1]
```diff
@@ -251,68 +259,53 @@ private void createFormLayoutWithBinder() {
...
         // E-mail and phone have specific validators
         Binding<Contact, String> emailBinding = binder.forField(email)
+                .withNullRepresentation("")
                 .withValidator(phoneOrEmailPredicate,
-                        "Both phone and email cannot be empty")
+                        "Please specify your email")
                 .withValidator(new EmailValidator("Incorrect email address"))
                 .bind(Contact::getEmail, Contact::setEmail);
 
         Binding<Contact, String> phoneBinding = binder.forField(phone)
```
[the remark this answers is not in the document]
Comment (anasmi) [reply vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java:294]: I am trying to simplify demo :D I don't think we should show all the binder capabilities in the demo for FormLayout. It's more about that it works with it

**Comment by anasmi** [comment]
@shadikhani what is about test? It runs locally on my machine and it also passes here, what is the issue you are having?

**Comment by shadikhani** [comment]
> @shadikhani what is about test? It runs locally on my machine and it also passes here, what is the issue you are having?

Yes, sorry. It was my ChromeDriver problem

File: vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (shadikhani) [reply vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java:294]: Ok

File: vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java [hunk 2]
```diff
@@ -251,68 +259,53 @@ private void createFormLayoutWithBinder() {
...
-                        "Please add the last name", 1, null))
-                .bind(Contact::getLastName, Contact::setLastName);
-
-        // Birthdate and doNotCall don't need any special validators
-        binder.bind(doNotCall, Contact::isDoNotCall, Contact::setDoNotCall);
-        binder.bind(birthDate, Contact::getBirthDate, Contact::setBirthDate);
+                        "Please add the address", 1, null))
+                .bind(Contact::getAddress, Contact::setAddress);
 
         // Click listeners for the buttons
```
[the remark this answers is not in the document]
Comment (anasmi) [reply vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java:310]: I would leave it as is. I don't think the change will improve the clarification (Not that it's very good now)

File: vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java [hunk 3]
```diff
@@ -120,23 +112,29 @@ public void setBirthDate(LocalDate birthDate) {
...
+            this.address = address;
+        }
+
         @Override
         public String toString() {
             StringBuilder builder = new StringBuilder();
             builder.append(firstName).append(" ").append(lastName);
             if (birthDate != null) {
-                builder.append(", born on ").append(birthDate);
+                builder.append("born on ").append(birthDate).append(", ");
```
Comment (tmattsso) [inline vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java:128]: missing a whitespace in the beginning, same for phone below if no birthdate, etc.

Comment (anasmi) [reply vaadin-form-layout-flow-demo/src/main/java/com/vaadin/flow/component/formlayout/demo/FormLayoutView.java:132]: Done. (Moved firstname and surname to the block. If they exist, than adding a space after there)

**Pull request merged**